### ISSUE TITLE
[QUESTION] Restarting `AIOKafkaConsumer` after `AIOKafkaConsumer.stop()`

### ISSUE THREAD

**Issue by dom-gunstone** [aio-libs/aiokafka#1010]
Hi,

I have a use case where I'm injecting a `AIOKafkaConsumer` instance into a class. The class has its own `start` and `stop` methods that amongst other things call the `start` and `stop` methods of the consumer class. The issue is that I'm failing the `assert self._fetcher is None` check in `AIOKafkaConsumer.start()` method.

Am I right in thinking this should work?

**Comment by ods** [comment]
No, consumer instance is not reusable, you have to recreate it.

**Comment by dom-gunstone** [comment]
@ods Thanks for confirming. 

Is this something that could be implemented, or is it not possible?

**Comment by ods** [comment]
It's possible to fix this, but I wouldn't expect somebody to invest their time in it.

**Comment by marcosschroh** [comment]
it is possible to fix it then it means that it should work as expected, unless that there is a clear decision of not to do it.

Can you explain the proper way of stopping a Consumer? I am asking because we have `stop()` and `unsubscribe()`. If the idea is to NOT reuse Consumers, then `stop()` should also call `unsubscribe()`, otherwise it does not make any sense.

If someone have the time to send a PR, would it be consider ?
